dev/src/index.ts

```diff
@@ -1345,6 +1364,39 @@ function validateDocumentData(
   return true;
 }
 
+/**
+ * Validates the use of 'options' as ReadOptions and enforces that 'fieldMask'
+ * is an array of strings or field paths.
+ *
+ * @private
+ * @param options.fieldMask - The subset of fields to return from a read
+ * operation.
+ */
+export function validateReadOptions(options: ReadOptions): boolean {
```

Comment:
Is there a reason this function returns a boolean? It looks like it either returns `true` or throws an error. I think either making it void and have it throw on error, or having it return `Error|null` and have the caller throw would be more consistent.